Mathlib/Init/Meta/WellFoundedTactics.lean

```diff
@@ -0,0 +1,48 @@
+/-
+Copyright (c) 2017 Microsoft Corporation. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Leonardo de Moura
+
+! This file was ported from Lean 3 source module init.meta.well_founded_tactics
+! leanprover-community/lean commit 855e5b74e3a52a40552e8f067169d747d48743fd
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+prelude
+import Mathlib.Init.Data.Nat.Lemmas
+
+-- Porting note: extremely partial port!
+
+-- TODO(Leo): move this lemma, or delete it after we add algebraic normalizer.
+-- TODO(Leo): move this lemma, or delete it after we add algebraic normalizer.
```

Comment:
```suggestion
```

I think it's time to retire the comment.

Reply:
Probably Gabriel meant for you also to delete the similar comments on the other lemmas?